bin/helpers/utils.js

```diff
@@ -1202,6 +1222,12 @@ exports.setConfig = (bsConfig, args) => {
 
 // blindly send other passed configs with run_settings and handle at backend
 exports.setOtherConfigs = (bsConfig, args) => {
+  if(isTestObservabilitySession() && process.env.BS_TESTOPS_JWT) {
+    bsConfig["run_settings"]["reporter"] = "browserstack-cypress-cli/bin/testObservability/reporter";
```

Comment:
the url can be added to config.js

Reply:
This is the path to our reporter. Moving to constants file.